pinot/regressors/biophysical_regressor.py

```diff
@@ -0,0 +1,78 @@
+# =============================================================================
+# IMPORTS
+# =============================================================================
+import torch
+import pinot
+import abc
+import math
+import numpy as np
+
+class BiophysicalRegressor(torch.nn.Module):
+    r""" Biophysically inspired model
+
+    Parameters
+    ----------
+
+    log_sigma : `float`
+        ..math:: \log\sigma observation noise
+
+    base_regressor : a regressor object that generates a latent F
+
+
+    """
+
+    def __init__(self, base_regressor=None):
+        super(BiophysicalRegressor, self).__init__()
+        self.base_regressor = base_regressor
+        self.log_sigma_measurement = torch.nn.Parameter(torch.zeros(1))
+
+
+	def g(self, func_value=None, test_ligand_concentration=1e-3):
```

Comment:
I feel like these should be a positional parameters. do you have other plans for it?

Reply:
elaborate ... I don't know what you mean.

Reply:
just that I think of keyword arguments as denoting something optional, but these [func_value, test_ligand_concentration] are not optional

Reply:
I see, it's a good distinction.